src/API/REST/V3/Routes/CURIE.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Give\API\REST\V3\Routes;
+
+use Give\Framework\Exceptions\Primitives\Exception;
+
+/**
+ * @unreleased
+ *
+ * WordPress version 4.5 introduced support for Compact URIs, or CURIEs. This makes it possible to
+ * reference links by a much simpler identifier than the full URL which could easily be quite lengthy.
+ *
+ * This will convert link URLs from https://api.mypluginurl.com/my_link` to my_plugin:my_linkin the API response.
+ * The full URL must still be used when adding links using WP_REST_Response::add_link() `.
+ */
+class CURIE
+{
+    /**
+     * @var string
+     */
+    private static $baseUrl = 'https://relations.givewp.com/';
```

Comment:
@jonwaldstein @glaubersilva We may want to coordinate with marketing to have a tentative subdomain and determine what it should be.

Reply:
I can help with that! what would happen if this subdomain wasn't live yet?

Reply:
@jonwaldstein Ideally, this page would be a doc resource to explain the givewp custom links relations, similar to this IANA page: https://www.iana.org/assignments/link-relations/link-relations.xhtml

However, we don't necessarily need to add content there for now. The WP core uses the `https://api.w.org/` base URL that redirects to the REST API Handbook page, but if you access a custom relation link like `https://api.wordpress.org/term`, you'll notice it's without any content as well - it returns a 404 error.